docs/UserGuide.md

```diff
@@ -224,9 +227,11 @@ In addition, there are some _optional_ extra parameters you can use to customize
   Example:`-since 21/10/2017`
 * **`-formats LIST_OF_FORMATS`**: A space-separated list of file extensions that should be included in the analysis. Default: `adoc cs css fxml gradle html java js json jsp md py tag xml`<br>
   Example:`-formats css fxml gradle`
+* **`-view`**: A flag to launch the report automatically after processing. Note that if an argument is given to this parameter, no analysis will be performed and the report specified by the argument will be opened.<br>
```

Comment:
should also add in the arg for the `-view`.
Here is the breakdown of each fields, hence needs to be more descriptive.

Reply:
This section of the UserGuide is under _Customizing the Analysis_, and is explaining optional extra parameters in addition to `-repo` parameter. Adding the description for the argument of `-view` here seems a bit out of place to me. There is another section describing the `-view` option called _Interpreting the Report_. Should I add it there instead?

Reply:
Sounds good to me.

Reply:
Actually this section [here](https://github.com/reposense/RepoSense/blob/release/docs/UserGuide.md#interpreting-the-report) already has some description for the argument of `-view`. Not quite sure what else I should add.

Reply:
hmm, I think in that case, let's just add in this section for completeness sake because the following line mentions about the argument. So perhaps, let's put ```**`-view [REPORT_FOLDER ]`**```, and update the message :

Note that if ~an~ `the REPORT_FOLDER` argument is given to this parameter,